common/sonar.js

```diff
@@ -11,11 +11,20 @@ function sonarStatusReferesher() {
 
     var rnd = Math.round(Math.random() * 1000000000000);
     var marker = pointURI.indexOf("&rnd=");
+    var wasPointRefreshedEarlier = marker !== -1
 
-    if (marker == -1) {
-      point.src = point.src + "&rnd=" + rnd.toString(26);
-    } else {
+    if (wasPointRefreshedEarlier) {
```

Comment:
I believe that it's prettier when the shortest/simplest if-branch comes first, so I've rewritten the if-else here a bit.